src/main/kotlin/org/jitsi/rtp/RtpHeader.kt

```diff
@@ -252,7 +252,7 @@ open class RtpHeader {
             appendln("hasExtension: $hasExtension")
             appendln("csrcCount: $csrcCount")
             appendln("marker: $marker")
-            appendln("payloadType: $payloadType")
+            appendln("packetType: $payloadType")
```

Comment:
was this change intentional? this is in rtp header

Reply:
Ops, I overdid it